README.md

```diff
@@ -1,36 +1,16 @@
 
-### Leading up to the first stable release
-*The following milestones lead up to the first release. During alpha and beta milestones, backwards compatibility is not guaranteed.  Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). Test cases will be added along the way. After RC-1 backward compatibility will be maintained*
-- [alpha-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-1) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-1.MD): Goal - Get feedback on the notation CLI client and notation user experience
-- [alpha-2](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-2) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-2.MD): Goal - Merge open PR on the Notaryv2 standard, CLI and Notation specifications 
-- [alpha-3](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-3) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-3.MD): Goal - Introduce Plugins for signing and verification. Manage signatures in OCI registries using [ORAS Artifacts Specification v1.0.0-RC.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2)
-- [alpha-4](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-4) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-4.MD): Goal - Add COSE signature format 
-- [beta-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ABeta-1)   [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.beta-1.MD): Goal - Verify using Trust Store and Trust Policy 
-- [RC-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-1)       [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.rc-1.MD)): Goal -First supported release from Notary v2 community. JWS and COSE signature formats. Notation CLI client 
-- [RC-2](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-2)       [TBD] Goal - Support "Fallback" method for OCI 1.0 based registries to manage signatures 
-
-| Repo | Related High-Level Roadmap items for above milestones  | Repo level project board (*filter based on milestones as you need them*)  |
-|------|-----------------------------------------------------------|--------------------------------------------------|
-|[Notaryproject](https://github.com/notaryproject/notaryproject)| [Issues labeled with "notaryproject"](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=label%3Anotaryproject) | [From all repos](https://github.com/orgs/notaryproject/projects/10)|
```

Comment:
This is confusing. Is this for the specifications or for overall questions about Notary Project? Also, let's point to the `specifications` repo.

Reply:
Those are removed lines, so no update needed.